### PR TITLE
Volunteer Application Disconnected UI

### PR DESCRIPTION
**Pull request by alexandrawang1**
# [Issue 159](https://github.com/MathSoc/mathsoc-website/issues/159)

This is the disconnected frontend part of volunteer applications. The form is submittable, but submitting does nothing at the moment. In the `router.post` function, you can access the full URL (including query parameters) as the "referer" header. When the backend part is being implemented, someone can parse the url based on how it's implemented.

This PR also updates the copy for many of the volunteer positions based on the current WordPress page.

Changes from the original form:
- no timezone
- no email opt-in
- "more info" section is no longer required

### CODE REVIEW

File: views/pages/get-involved/volunteer-application.pug [hunk 1]
```diff
@@ -0,0 +1,52 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+  link(href="/assets/css/pages/get-involved/volunteer-application.css" rel="stylesheet")
+
+block body 
+  div(class="content")
+    +section("white")
+      form(action="/api/volunteer-application" method="post")
+        label(for="first-name" class="required") First Name
```
Comment (some-molasses) [inline views/pages/get-involved/volunteer-application.pug:10]: loving the accessible tags!!

File: views/pages/get-involved/volunteer-application.pug [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          each term in data.terms
+            option(value=term) #{term}
+
+        p(class="required") For the term that you are looking to volunteer are you on co-op?
+        div(class="radio-select")
+          input(id="coop-true" type="radio" name="coop" required)
+          label(for="coop-true") Yes
+        div(class="radio-select")
+          input(id="coop-false" type="radio" name="coop" required)
+          label(for="coop-false") No s
```
Comment (some-molasses) [inline views/pages/get-involved/volunteer-application.pug:40]: typo?

File: server/data/get-involved/volunteer-application.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "Actuarial Science",
+    "Applied Mathematics",
+    "Combinatorics and Optimization",
+    "Computational Mathematics",
+    "Computer Science",
+    "Computing and Financial Management",
+    "Double Degree (CS)",
+    "Double Degree (Math)",
+    "Financial Analysis and Risk Management",
+    "Math Teaching",
```
Comment (some-molasses) [inline server/data/get-involved/volunteer-application.json:13]:
```suggestion
    "Math/Teaching",

```

**Review by some-molasses — APPROVED** [review]
looks (99%) good!!

File: server/data/get-involved/volunteer-application.json (hunk 3 above)
Comment (alexandrawang1) [reply server/data/get-involved/volunteer-application.json]: :done:

File: views/pages/get-involved/volunteer-application.pug (hunk 2 above)
Comment (alexandrawang1) [reply views/pages/get-involved/volunteer-application.pug]: yes 😅

**Review by ad-world — APPROVED** [review]

**Review by ad-world — APPROVED** [review]

**Pull request merged**